src/store/elementalChat.js

```diff
@@ -104,14 +104,27 @@ export default {
     channels: [],
     currentChannelId: null,
     stats: {},
-    statsLoading: false
+    statsLoading: false,
+    agentHandle: '',
+    needsHandle: false
```

Comment:
Possible clarifying comment if it's not possible to just get rid of this field
```suggestion
    // If needsHandle is true then we prompt the user to enter a handle
    needsHandle: false
```